File: LeetCode/Easy/344-reverse-string/try-1.java
/*
 * Problem #344: Reverse String
 * Difficulty: Easy
 * Submission: Try 1
 * status: Accepted
 * Language: java
 * Date: 7/10/2026, 1:23:04 PM
 * Link: https://leetcode.com/problems/reverse-string/
 */

class Solution {
    public void reverseString(char[] s) {
        int left = 0, right = s.length - 1;

        while (left < right) {
            char temp = s[left];
            s[left] = s[right];
            s[right] = temp;

            left++;
            right--;
        }
    }
    }

File: Codeforces/158/A/try-1.py
/*
 * Problem : Next Round
 * Difficulty: Easy
 * Submission: Try 1
 * status: Accepted
 * Language: Python 3
 * Date: 7/28/2026, 7:12:40 PM
 * Link: https://codeforces.com/contest/158/problem/A
 */

n, k = map(int, input().split())
scores = list(map(int, input().split()))

cutoff = scores[k - 1]

count = 0
for score in scores:
    if score >= cutoff and score > 0:
        count += 1

print(count)
